packages/zenon_modulo/zenon_modulo.0.5.0/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "2.0"
+homepage: "https://github.com/Deducteam/zenon_modulo"
+dev-repo: "git+https://github.com/Deducteam/zenon_modulo.git"
+maintainer: "Guillaume Burel <guillaume.burel@ensiie.fr>"
+authors: [ "INRIA and contributors" ]
+license: "BSD-3-Clause"
+bug-reports: "https://github.com/Deducteam/zenon_modulo/issues"
+tags: [ "automated theorem prover" ]
+depends: [
+  "ocaml" {>= "4.08.0"}
+  "zarith" {>= "1.11"}
+]
+depopts: [
+  "coq"
+]
+build: [
+  ["./configure"
+   "--prefix" prefix
+   "--libdir" lib]
+  [make]
+]
+install: [[make "install"]]
+synopsis: "Zenon Modulo Theory"
```

Comment:
```suggestion
conflicts: ["ocaml-option-bytecode-only"]
synopsis: "Zenon Modulo Theory"
```
This will solve
```
# Cannot find ocamlopt[.opt] compiler in path
```